diff --git a/dist/index.js b/dist/index.js
index 97b00aa194cfacd383e71bb9a2bfd44d483c9895..4c3a6ebe2e17a3289983d2b926cae91c44439a28 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1687,9 +1687,7 @@ import path7 from "node:path";
 import react from "@vitejs/plugin-react";
 import fs6 from "fs-extra";
 import normalizePath3 from "normalize-path";
-import {
-  splitVendorChunkPlugin
-} from "vite";
+var splitVendorChunkPlugin = () => ({ name: "split-vendor-chunk-noop" });
 
 // src/next/vite/cors.ts
 var LOCALHOST_RE = /^https?:\/\/(?:localhost|127\.0\.0\.1|\[::1\])(:\d+)?$/;
